includes/class-llms-blocks-assets.php

```diff
@@ -198,14 +200,33 @@ private function use_bc_assets() {
 	 *
 	 * @since 2.4.3
 	 *
-	 * @link https://github.com/gocodebox/lifterlms-blocks/issues/170
+	 * @return bool
+	 * @link  https://github.com/gocodebox/lifterlms-blocks/issues/170
 	 *
-	 * @return boolean
 	 */
-	private static function can_variation_transform_icon_be_an_object() {
+	private static function can_variation_transform_icon_be_an_object(): bool {
 		global $wp_version;
+
 		return version_compare( $wp_version, '6.0-src', '<' ) && ! defined( 'GUTENBERG_VERSION' )
-				|| ( defined( 'GUTENBERG_VERSION' ) && version_compare( GUTENBERG_VERSION, '13.0', '<' ) );
+			   || ( defined( 'GUTENBERG_VERSION' ) && version_compare( GUTENBERG_VERSION, '13.0', '<' ) );
```

Comment:
```suggestion
			   || ( defined( 'GUTENBERG_VERSION' ) && version_compare( GUTENBERG_VERSION, '13.0', '<' ) );
```
we don't want to use spaces for indentation, could you please use tabs only?